lib/utils/fs/readFileIfExists.test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const path = require('path');
+const chai = require('chai');
+const readFileIfExists = require('./readFileIfExists');
+
+// Configure chai
+chai.use(require('chai-as-promised'));
+chai.use(require('sinon-chai'));
+const expect = require('chai').expect;
+
+describe('#readFileIfExists()', () => {
+  it('should resolve with file content if file exists', () => {
+    const packageJsonFilePath = path.join(__dirname, '..', '..', '..', 'package.json');
```

Comment:
Although this ships with Serverless I'd say that relying on external files in tests (which are not generated by the test framework itself) is not idea since the test will break as soon as the outer environment changes.

A possible solution would be to stub the `readFile` call or use an tmp file.